packages/components/src/components/message/message.tsx

```diff
@@ -7,8 +7,8 @@ import { cn } from '~/lib/utils';
 const messageVariants = cva('flex w-full gap-x-2.5 justify-start p-3 text-base', {
   variants: {
     variant: {
-      success: 'bg-green-300/20 [&>svg]:text-green-300',
-      error: 'bg-red-100/20 [&>svg]:text-red-100',
+      success: 'bg-success-light/[.15] [&>svg]:text-success',
+      error: 'bg-error/[.15] [&>svg]:text-error-secondary',
```

Comment:
Related to my other comment, this seems like a one off and we should probably remove.

Reply:
oo sory will remove those lines. maybe I can include them on a separate pull regarding the message component. it does not uses standart color and it missing 3 variant. 
<img width="997" alt="Screenshot 2024-03-07 at 01 18 28" src="https://github.com/bigcommerce/catalyst/assets/16627430/1a299c41-2c3d-496c-8072-61f9d9e012e1">

Reply:
Let me confirm with design and I'll reach back, thanks!

Reply:
I confirmed we should stick to the changes below so we can get rid of `bg-success-light`.

```suggestion
      success: 'bg-success-secondary/15 [&>svg]:text-success',
      error: 'bg-error-secondary/15 [&>svg]:text-error',
```

Reply:
done on commit `Change Message Component Background Colors Based on Design Comment`